### PR TITLE
Minor Fix And Command Word Rename

### PR DESCRIPTION
**Pull request by lirc572**

### CODE REVIEW

**Review by samuelfangjw — APPROVED** [review]
looks good

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -46,12 +46,12 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
     }
 
     /**
-     * Parses a {@code String name} into a {@code Name}.
+     * Parses a {@code String name} into a {@code seedu.address.model.contact.Name}.
```
Comment (vevek) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:49]: Should just be name here. Its showing the entire package currently

**Review by vevek — CHANGES_REQUESTED** [review]
Just the minor comment!

**Review by vevek — APPROVED** [review]
LGTM

**Pull request merged**